crates/mitex/src/lib.rs

```diff
@@ -274,6 +360,50 @@ impl MathConverter {
                 let name = name.text();
                 // remove prefix \
                 let name = &name[1..];
+
+                // hack for \item in itemize and enumerate
+                if name == "item" {
+                    if matches!(self.env, LaTeXEnv::Itemize | LaTeXEnv::Enumerate) {
+                        f.write_char('\n')?;
+                        for _ in 0..(self.indent - 2) {
+                            f.write_char(' ')?;
+                        }
+                        if matches!(self.env, LaTeXEnv::Itemize) {
```

Comment:
similarly, we can have a helper to make conditional output, which doesn't change performance at all, but improves reability.